tools/build_api.py

```diff
@@ -235,7 +235,7 @@ def build_project(src_path, build_path, target, toolchain_name,
 
             add_result_to_report(report, cur_result)
 
-        return res
+        return res, toolchain.map_outputs
```

Comment:
Yes, I missed `add_result_to_report()` completely. I will add this to this functionality and revert changes to `build_project()` API.